tests/test_apigateway/test_apigateway_validators.py

```diff
@@ -0,0 +1,168 @@
+    client = create_client()
+    api_id = create_rest_api_id(client)
+    response = create_validator(client, api_id)
+    response.pop(RESPONSE_METADATA)
+    response.pop(ID)
+    response.should.equal(
+        {
+            NAME: PARAM_NAME,
+            VALIDATE_REQUEST_BODY: True,
+            VALIDATE_REQUEST_PARAMETERS: True,
+        }
+    )
+
+
+@mock_apigateway
+def test_get_request_validators():
+
+    client = create_client()
+    api_id = create_rest_api_id(client)
+    response = client.get_request_validators(restApiId=api_id)
+
+    validators = response["items"]
+    validators.should.have.length_of(0)
+
+    response.pop(RESPONSE_METADATA)
+    response.should.equal({"items": []})
+
+    response = create_validator(client, api_id)
+    validator_id1 = response[ID]
+    response = create_validator(client, api_id)
+    validator_id2 = response[ID]
+    response = client.get_request_validators(restApiId=api_id)
+
+    validators = response["items"]
+    validators.should.have.length_of(2)
+
+    response.pop(RESPONSE_METADATA)
+    response.should.equal(
+        {
+            "items": [
+                {
+                    ID: validator_id1,
+                    NAME: PARAM_NAME,
+                    VALIDATE_REQUEST_BODY: True,
+                    VALIDATE_REQUEST_PARAMETERS: True,
+                },
+                {
+                    ID: validator_id2,
+                    NAME: PARAM_NAME,
+                    VALIDATE_REQUEST_BODY: True,
+                    VALIDATE_REQUEST_PARAMETERS: True,
+                },
+            ]
+        }
+    )
+
+
+@mock_apigateway
+def test_get_request_validator():
+    client = create_client()
+    api_id = create_rest_api_id(client)
+    response = create_validator(client, api_id)
+    validator_id = response[ID]
+    response = client.get_request_validator(
+        restApiId=api_id, requestValidatorId=validator_id
+    )
+    response.pop(RESPONSE_METADATA)
+    response.should.equal(
+        {
+            ID: validator_id,
+            NAME: PARAM_NAME,
+            VALIDATE_REQUEST_BODY: True,
+            VALIDATE_REQUEST_PARAMETERS: True,
+        }
+    )
+
+
+@mock_apigateway
+def test_delete_request_validator():
+    client = create_client()
+    api_id = create_rest_api_id(client)
+    response = create_validator(client, api_id)
+    # test get single validator by
+    validator_id = response[ID]
+    response = client.get_request_validator(
+        restApiId=api_id, requestValidatorId=validator_id
+    )
+
+    response.pop(RESPONSE_METADATA)
+    response.should.equal(
+        {
+            ID: validator_id,
+            NAME: PARAM_NAME,
+            VALIDATE_REQUEST_BODY: True,
+            VALIDATE_REQUEST_PARAMETERS: True,
+        }
+    )
+
+    # delete validator
+    response = client.delete_request_validator(
+        restApiId=api_id, requestValidatorId=validator_id
+    )
+    with pytest.raises(RequestValidatorNotFound):
```

Comment:
Sorry, I should have been more explicit in the first place. 
Negative tests should be in the following format:
```
from botocore.exceptions import ClientError
with pytest.raises(ClientError) as ex:
    client.get_request_validator(
        restApiId=api_id, requestValidatorId=validator_id
    )
err = ex.value.response["Error"]
err["Code"].should.equal("NotFoundException")
err["Message"].should.equal("Invalid Request Validator Id specified")
```
This verifies that the exception is formatted correctly. See my other comment on adding the `self.error()` back in.

Changing this will fix the ServerMode test that is currently failing.